server/rooms.ts

```diff
@@ -693,7 +693,7 @@ export abstract class BasicRoom {
 		// add an alias from the old id
 		Rooms.aliases.set(oldID, newID);
 		if (!this.settings.aliases) this.settings.aliases = [];
-		this.settings.aliases.push(oldID);
+		if (!this.settings.aliases.includes(oldID)) this.settings.aliases.push(oldID);
```

Comment:
In theory, this should never happen. Did you manage to get it to happen somehow?

Reply:
yes - if `/renameroom` was used while the title-saving bug was in effect, an alias would be added for the "old" name, but then the old name is restored on restart.

Reply:
We can probably remove the line if this hasn't happened to any rooms on main, though.

Reply:
```suggestion
		// resolve an old (fixed) bug in /renameroom
		if (!this.settings.aliases.includes(oldID)) this.settings.aliases.push(oldID);
```

Reply:
Oh, that's fine, although it could use comment documentation.